### PR TITLE
Add installation instructions for NixOS

### PR DESCRIPTION
**Pull request by JohnAZoidberg**
I thought some documentation on how to install/use py3status on NixOS would be nice since it took me quite some time to figure out.
NixOS is a mature distribution with a small but steadily growing userbase but I think it has a lot of potential and it is one of the fastest moving repositories on [GitHub](https://github.com/nixos/nixpkgs).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -117,6 +117,26 @@ Using dnf:
 
     $ dnf install py3status
 
+NixOS
+----------
+Add py3status as a Python 3.6 package with
```
Comment (lasers) [inline README.rst]: No love for `3.7`? Can `nixOS` haz `python2.7-py3status-3.7`, `python3.6-py3status-3.7` too?

Comment (JohnAZoidberg) [reply README.rst]: 3.6 is the python version ;)
py3status3.7 is already available for NixOS: NixOS/nixpkgs/pull/32490

Comment (lasers) [reply README.rst]: Conflict.

**Comment by tobes** [comment]
It would be nice to add this to https://py3status.readthedocs.io/en/latest/intro.html#installation too, but I think we need to change from the table layout to do so.

**Comment by JohnAZoidberg** [comment]
I added short instructions on how to do it on the commandline with a link to the longer configuration in the README. I think the table should still be ok.

**Comment by tobes** [comment]
Sorry you've a merge conflict due to updates in the debian/ubuntu install instructions.

This is also why tables are a pain

File: doc/intro.rst [hunk 2]
```diff
@@ -41,25 +41,25 @@ Philosophy
...
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**Gentoo Linux**   |``$ emerge -a py3status``               |                                                                            |
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**Pypi**           |``$ pip install py3status``             |                                                                            |
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**Void Linux**     |``$ xbps-install -S py3status``         |                                                                            |
++-------------------+----------------------------------------+----------------------------------------------------------------------------+
+|**NixOS**          |``$ nix-env -i python3.6-py3status-3.7``|Not global. See `README <https://github.com/ultrabug/py3status#nixos>`_     |
```
Comment (lasers) [inline doc/intro.rst]: Suggestion: Keep it simple. `Not global. See README.rst.` And fix the `bstable` typo.

**Comment by ultrabug** [comment]
Thanks @JohnAZoidberg ! 

Like my fellows, I would like the table to not be broken like this, please just make it so you add your own row on the diff?

**Comment by lasers** [comment]
@JohnAZoidberg Please address @ultrabug concern. Thanks. :-)

P.S. I think you can use `[\?not_zero {swap_used}+]` to make it disappear instead of `0.0+`.

**Comment by tobes** [comment]
Hi looking much cleaner.

There are some merge conflicts, could you rebase and fix them thanks.  Then I'm happy to merge this

**Comment by lasers** [comment]
@tobes Hi friend. A gentle bump on this PR.

**Comment by tobes** [comment]
@JohnAZoidberg thanks sorry it's taken so long to get this merged

**Pull request merged**